excellent/functions/builtin.go

```diff
@@ -1628,6 +1629,25 @@ func Unique(env envs.Environment, array *types.XArray) types.XValue {
 	return types.NewXArray(unique...)
 }
 
+// Append returns the result of appending one array to another as a new array.
+//
+//   @(append(array("a", "b"), array("c", "d"))) -> [a, b, c, d]
```

Comment:
I think the second argument could be an element to append and not array 
or probably we rename the function to concat instead of append

`@(append(array("a", "b"), "c")) -> [a, b, c]`

Reply:
`@(concat(array("a", "b"), array("c", "d"))) -> [a, b, c, d]`

Reply:
I was thinking that `concat` sounded too much like a string operation but now I see Javascript uses `concat` for arrays so that's perfect https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat

Re adding single elements.. since you can just the single element with `array()`.. not feeling that's required.